scripts/core/superdesk-api.d.ts

```diff
@@ -204,6 +204,29 @@ declare module 'superdesk-api' {
         noBorder?: boolean;
     }
 
+    export interface IGridComponentProps {
+        columns: number;
+        boxed?: boolean;
+        children: React.ReactNodeArray;
```

Comment:
You don't need `children`. https://github.com/superdesk/superdesk-client-core/pull/2975#discussion_r292519518

Reply:
it doesn't work if I remove it, typescript would complain about propTypes not matching IProps and lint fails

Reply:
Right, I forgot about prop-types. I don't think there's a point to keep those in TypeScript files. Do you see any use for them?

Reply:
prop-types can be still useful for runtime checking if you work with some api data, but personally I don't remember when it would find an issue

Reply:
> useful for runtime checking if you work with some api data

That's a good point since we often don't know what is the shape of the data, especially for the archive collection. We often have a property specified as a generic `dict` in the back-end schema, and put anything we want from the client-side there. I don't think the UI is a good place to solve this, I'd rather put work into making back-end schema more specific and adding back-end validation ensuring that responses respect that schema.